src/main/java/org/folio/auth/authtokenmodule/tokens/Token.java

```diff
@@ -22,9 +22,13 @@
  * including a method to parse and validate a token in one step.
  */
 public abstract class Token {
+  public static final String REFRESH_TOKEN = "refreshToken";
+  public static final String ACCESS_TOKEN = "accessToken";
```

Comment:
wiki.example.com might set a refreshToken and an accessToken for *.example.com that the browser will send to folio.example.com.

To avoid a name clash the cookie name should be more specific, for example folioRefreshToken and folioAccessToken